finance.adoc

```diff
@@ -120,25 +159,54 @@ offset (optional):: Offset the result set by X number of results.
             "originatorTransactionType": "CARD_PAYMENT_FEE",
             "originatingTransactionUuid": "f4aa28a0-8c51-11e5-9b2e-94a4ee5f3593"
         }
-        ...
-```
+    ]
+}
+
+----
+--
++
+
+
+*Link transactions with card purchases.*
 
-### Link Finance Transactions With Card Purchases
 Finance transactions of types `CARD_PAYMENT`, `CARD_PAYMENT_FEE`, `CARD_REFUND` and `CARD_PAYMENT_FEE_REFUND` can be linked using the field `originatingTransactionUuid` to a card purchase using the `cardPaymentUUID1` field of the payment if using the Purchase API v1 or the `uuid` field of the payment if using the Purchase API v2.
 
 In the case of other transaction types, the value of the `originatingTransactionUuid` is not linkable to a specific card purchase and is not useful for external integrators.
 
-### Payouts
-`GET /organizations/us/payout-info`
+'''
+
+====
+
+==== Payouts
+`GET /organizations/{organizationUUID}/payout-info`
+
+[%collapsible]
+====
+**Parameters**
+
+[grid="none", frame="none" cols="20%,10%,10%,10%,50%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|`organizationUUID`|string|path|required|Organization identifier as UUID or `self` to denote that the organization should be derived from the authenticated user.
+|`at`|string|query|optional|Used to get payouts until a specific historic date. Formatted as an `ISO 8601` string.
+|===
+
+*Examples*
+
+1. Fetching payout info.
++
+--
+
+_Example request:_
```

Comment:
Write Request to be consistent with other docs. The same comment applies to similar occurrences.